consensus/src/dag/dag_fetcher.rs

```diff
@@ -19,44 +19,51 @@ use tokio::sync::{
     oneshot,
 };
 
-enum FetchCallback {
+pub enum LocalFetchRequest {
```

Comment:
This is more like FetchState than a Request no?

Reply:
in a sense, it's a request sending from local component like dag_driver